### ISSUE TITLE
Some icons are not dismissing themselves after the mouse is hovering over a new icon

### ISSUE THREAD

**Issue by Gonzom** [Js41637/Overwatch-Item-Tracker#235]
Some icons are not dismissing themselves after the mouse is hovering over a new icon.
This _might_ only happen with "default" icons, as I couldn't reproduce this with other ones.
See attached screenshots.

![dups1](https://user-images.githubusercontent.com/16445220/40532517-5f480272-6008-11e8-9024-71837116ba26.png)
![dups2](https://user-images.githubusercontent.com/16445220/40532518-5f702130-6008-11e8-83f0-db1fcb8d8c5e.png)

Steps to reproduce:
1. Go to "All Class".
2. Hover over a "default" icon.
3A. If layout has 2 columns of icons - hover over a different icon from a different column.
3B. If layout has 1 columns of icons - hover over a spray.
4. Previous icon should still be showing.

Using Chrome on Windows.

**Comment by Js41637** [comment]
Yeah I assume it has to do with the mousing over the disabled inputs and its preventing events from being triggered. tbh I don't think there is anything I can do to fix it I don't really have any control over the the tooltips.
